f2/src/forms/ContactInfoFormSchema.js

```diff
@@ -57,26 +47,58 @@ const createErrorSummary = (errors) => {
     }
   }
 
-  if (email) {
-    errorSummary['email'] = {
-      label: contactFormOptions.email.label,
-      message: contactFormOptions.email.errorMessage,
+  if (emailORphone) {
+    errorSummary['emailORphone'] = {
+      label: contactFormOptions.emailORphone.label,
+      message: contactFormOptions.emailORphone.errorMessage,
     }
   }
 
-  if (phone) {
-    errorSummary['phone'] = {
-      label: contactFormOptions.phone.label,
-      message: contactFormOptions.phone.errorMessage,
-    }
+  return errorSummary
+}
+
+const fyiValidate = (values) => {
+  const errors = {}
+  const fields = {}
```

Comment:
Why is a field object required?

Reply:
Solved, no longer use this, new algorism implemented